BasicSample/Assets/ARAnchor/Scripts/SampleAnchor.cs

```diff
@@ -39,26 +42,37 @@ public bool Persisted
             set
             {
                 m_persisted = value;
-                meshRenderer.material = m_persisted ? persistentAnchorMaterial : transientAnchorMaterial;
+                meshRenderer.material = m_trackingState == TrackingState.Tracking ? (m_persisted ? persistentAnchorMaterial : transientAnchorMaterial) : untrackedAnchorMaterial;
+                UpdateText();
+            }
+        }
+
+        private TrackingState m_trackingState = TrackingState.None;
+        public TrackingState TrackingState
+        {
+            get => m_trackingState;
+            set
+            {
+                m_trackingState = value;
+                meshRenderer.material = m_trackingState == TrackingState.Tracking ? (m_persisted ? persistentAnchorMaterial : transientAnchorMaterial) : untrackedAnchorMaterial;
                 UpdateText();
             }
         }
 
         private void UpdateText()
         {
-            if (m_arAnchor == null) return;
-            text.text = (Persisted ? $"\"{m_name}\": " : "") + m_arAnchor.trackableId.ToString();
+            if (m_arAnchor == null || text == null) return;
+            text.text = (Persisted ? $"\"{Name}\": " : "") + m_arAnchor.trackableId.ToString() + "\nTracking State: " + TrackingState;
```

Comment:
can you do me a favor to move the persisted name to the 2nd line? so it will be const width like this
> c14abc75-f0d6-4382-b308-b57e65ac24a5
> persisted name, tracking state.

Reply:
Updated!